packages/auth/amplify_auth_cognito_dart/lib/src/state/machines/sign_in_state_machine.dart

```diff
@@ -414,6 +419,11 @@ class SignInStateMachine extends StateMachine<SignInEvent, SignInState> {
         })
         ..clientMetadata.addAll(event.clientMetadata);
 
+      if (config.pinpointEndpointId != null) {
```

Comment:
All `InitiateAuthRequest`s and `RespondToAuthChallengeRequest`s should have this, but some don't.

Reply:
Thanks added